builtin/async/mainmenu.lua

```diff
@@ -1,9 +1,22 @@
 core.log("info", "Initializing asynchronous environment")
 
+
 function core.job_processor(func, serialized_param)
 	local param = core.deserialize(serialized_param)
 
 	local retval = core.serialize(func(param))
 
 	return retval or core.serialize(nil)
 end
+
+
+function core.get_accept_languages()
```

Comment:
1. Shouldn't this function be `local` in `contentdb.lua`? There is currently no other code that would make use of this function.
2. `http_get_accept_language_header` might be a more fitting name.

Reply:
I believe this is here because locals aren't accessible from the async environment, and i want this loaded into the async environment